src/interfaces/stepper-interfaces/AddPeopleAndRole/people-and-roles-resource-interface.ts

```diff
@@ -0,0 +1,19 @@
+import { Rules } from '@/enums'
+
+interface Rule {
+  rule: Rules
+  text: string
+  test: (...args) => boolean // rule function
+}
+
+export interface PeopleAndRolesResourceIF {
+  header: string
+  blurb: string
+  helpSection: {
+    header: string
+    helpText: Array<string>
+  }
+  addIncorporator: boolean
+  addOrganization: boolean
+  rules: Array<Rule>
+}
```

Comment:
Note that I didn't make any of these properties optional, which forces the resource files to declare all fields. I feel that making properties optional implements a "main value" and optional "custom value" (ie, inheritance), which is something I think we should try to avoid, since it may become confusing whether a particular filing is using its custom value or an inherited value. So my chosen approach leads to some duplication in the resource files (ie, several entity types have the same values), but I think this will avoid future confusion. Thoughts?